opennms-webapp/src/main/webapp/includes/help-software-management.jsp

```diff
@@ -0,0 +1,52 @@
+<%--
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+--%>
+
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Software Management</h3>
+    </div>
+    <div class="panel-body">
+        <table class="table">
+            <tr>
+                <td style="border-top: none;"><a href="https://github.com/OpenNMS/opennms" target="_blank" class="btn btn-default" role="button" style="width: 100%">OpenNMS on GitHub</a></td>
+                <td style="border-top: none;">Our source code is version controlled with git and hosted on GitHub. Download the source code from latest develop or released versions of OpenNMS.</td>
```

Comment:
Change:
`Download the source code from latest develop or released versions of OpenNMS.`
to
`Download the source code for released versions of OpenNMS or the latest development code.`